src/index.html

```diff
@@ -6,9 +6,45 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="#page_top">
+    <header>
+      <a href="#page_top"></a><img src="images/logo.png" alt="moyo"></a>
+      <nav class="nav__items">
+        <li class="li__item">
+          <a
+            class="is-active"
+            href="index.html">APPLE</a>
+        </li>
+        <li class="li__item">
+          <a href="index.html">SAMSUNG</a>
+        </li>
+        <li class="li__item">
+          <a href="index.html">SMARTPHONES</a>
+        </li>
+        <li class="li__item">
+          <a
+            data-qa="hover"
+            class="a__item"
+            href="index.html">LAPTOPS & COMPUTERS</a>
+        </li>
+        <li class="li__item">
+          <a href="index.html">GADGETS</a>
+        </li>
+        <li class="li__item">
+          <a href="index.html">TABLETS</a>
+        </li>
+        <li class="li__item">
+          <a href="index.html">PHOTO</a>
+        </li>
+        <li class="li__item">
+          <a href="index.html">VIDEO</a>
+        </li>
```

Comment:
```suggestion
        <li class="li__item">
          <a
            class="is-active"
            href="index.html">APPLE</a>
        </li>
        
        <li class="li__item">
          <a href="index.html">SAMSUNG</a>
        </li>
        
        <li class="li__item">
          <a href="index.html">SMARTPHONES</a>
        </li>
        
        <li class="li__item">
          <a
            data-qa="hover"
            class="a__item"
            href="index.html">LAPTOPS & COMPUTERS</a>
        </li>
        
        <li class="li__item">
          <a href="index.html">GADGETS</a>
        </li>
        
        <li class="li__item">
          <a href="index.html">TABLETS</a>
        </li>
        <li class="li__item">
          <a href="index.html">PHOTO</a>
        </li>
        
        <li class="li__item">
          <a href="index.html">VIDEO</a>
        </li>
```

Wrap your list items with <ul> tab https://www.w3schools.com/html/html_lists.asp

Also add blank line between multiple sibling blocks of html